### PR TITLE
testing/bup: new aport

### PR DESCRIPTION
**Pull request by adamruzicka**
and dependency

TODO:
- [ ] implement `check`

### CODE REVIEW

File: testing/py-xattr/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Maintainer: Adam Ruzicka <a.ruzicka@outlook.com>
+pkgname=py-xattr
+_pkgname=pyxattr
+pkgver=0.6.0
+pkgrel=0
+pkgdesc="A python module for manipulating extended attributes"
+url="https://pyxattr.k1024.org/"
+arch="all"
+license="GPL"
```
Comment (tanertas) [inline testing/py-xattr/APKBUILD]: Alpine uses spdx license format. Please pick proper one from https://spdx.org/licenses/

File: testing/py-xattr/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+depends="python2"
+depends_dev=""
+makedepends="python2-dev attr-dev"
+install=""
+subpackages=""
```
Comment (tanertas) [inline testing/py-xattr/APKBUILD]: You may remove unused variables like three above.

File: testing/py-xattr/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+source="https://pyxattr.k1024.org/downloads/pyxattr-$pkgver.tar.gz"
+
+_builddir="$srcdir"/$_pkgname-$pkgver
```
Comment (tanertas) [inline testing/py-xattr/APKBUILD]: No need custom _builddir here. You just could override $builddir and use everywhere when need it.

File: testing/py-xattr/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+build() {
+	cd "$_builddir"
+	python2 setup.py build || return 1
```
Comment (tanertas) [inline testing/py-xattr/APKBUILD]: " || return 1" is not used anymore. You may discard all of them.

**Comment by Ikke** [comment]
Checks are currently failing. You might want to rebase this so that you get drone integration as well.

**Comment by adamruzicka** [comment]
I'm afraid python 3 support is in the works but not done yet (see https://github.com/bup/bup/commit/d30215975cb04ed75f027006defa5b35c3f6a3ac). So bup itself needs python2 and therefore it needs py-xattr for python2 as well

**Pull request closed without merging**